data/initrd/scripts/url/hmc/menu

```diff
@@ -0,0 +1 @@
+HMC
```

Comment:
You have to prepare your machine specifically to make use of the HMC option. So you'd hardly miss the entry.